components/net/http_loader.rs

```diff
@@ -1447,10 +1455,7 @@ fn http_network_fetch(
         &request.method,
         &request.headers,
         request.body.as_mut().and_then(|body| body.take_stream()),
-        request
-            .body
-            .as_ref()
-            .map_or(0, |body| body.len().unwrap_or(0)),
+        request.body.as_ref().map_or(None, |body| body.len()),
```

Comment:
This can be `request.body.as_ref().map(|body| body.len())`

Reply:
addressed!